packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -4448,6 +4571,11 @@ class SemanticsConfiguration {
   ///  * [addTagForChildren] to add a tag and for more information about their
   ///    usage.
   Iterable<SemanticsTag>? get tagsForChildren => _tagsForChildren;
+
+  /// Whether the this configuration will tag the child semantics nodes with a
+  /// given [SemanticsTag].
+  bool isChildrenTagged(SemanticsTag tag) => _tagsForChildren?.contains(tag) ?? false;
```

Comment:
maybe rename to: `tagsChildrenWith`, `isChildrenTagged` doesn't sound like correct english.